app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -839,6 +806,94 @@ protected void setupPoll(Poll poll, List<Emoji> emojis, StatusActionListener lis
 
             pollDescription.setText(pollInfo);
 
+
+        }
+    }
+
+    private void setupPollResult(Poll poll, List<Emoji> emojis) {
+        List<PollOption> options = poll.getOptions();
+
+        for(int i = 0; i < Status.MAX_POLL_OPTIONS; i++) {
+            if(i < options.size()) {
```

Comment:
Here and in all other places, can we get a crash if the server supports more options for some reason?